src/applications/mhv/secure-messaging/containers/FolderThreadListView.jsx

```diff
@@ -253,6 +253,7 @@ const FolderThreadListView = props => {
           <>
             <FolderHeader
               folder={folder}
+              threadCount={threadList}
```

Comment:
if this is a `threadCount` which I guess is a number, do we want to pass `threadList.length` instead?

Reply:
This is updated.